packages/fxa-payments-server/src/index.scss

```diff
@@ -1,3801 +1 @@
-    margin-bottom: 30px;
-}
-html[dir=ltr] .fxa-radio {
-    text-align: left;
-}
-html[dir=rtl] .fxa-radio {
-    text-align: right;
-}
-.fxa-radio label {
-    display: block;
-}
-.email-list {
-    padding: 0;
-}
-.email-list .email-address {
-    height: 40px;
-    list-style: none;
-    margin: 10px 0;
-    position: relative;
-}
-html[dir=ltr] .email-list .email-address {
-    background-position: left 2px;
-}
-html[dir=rtl] .email-list .email-address {
-    background-position: right 2px;
-}
-.email-list .email-address .address {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-}
-@media screen and (min-width:400px) {
-    .email-list .email-address .address {
-        width: calc(95% - 95px);
-    }
-}
-@media screen and (max-width:400px) {
-    .email-list .email-address .address {
-        width: 100%;
-    }
-}
-@media screen and (min-width:400px) {
-    .email-list .email-address .address.verified {
-        width: calc(95% - 195px);
-    }
-}
-.email-list .email-address .settings-button-group {
-    position: absolute;
-    top: 0;
-}
-html[dir=ltr] .email-list .email-address .settings-button-group {
-    right: 0;
-}
-html[dir=rtl] .email-list .email-address .settings-button-group {
-    left: 0;
-}
-.email-list .email-address .settings-button-group .settings-button {
-    display: inline-block;
-    height: 35px;
-    margin-left: 10px;
-    min-width: 100px;
-    overflow: hidden;
-    text-align: center;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    width: 100px;
-}
-@media screen and (max-width:400px) {
-    .email-list .email-address .settings-button-group {
-        display: none;
-    }
-}
-.email-list .email-options {
-    height: 40px;
-    list-style: none;
-    margin: 10px 0;
-    position: relative;
-}
-html[dir=ltr] .email-list .email-options {
-    background-position: left 2px;
-}
-html[dir=rtl] .email-list .email-options {
-    background-position: right 2px;
-}
-.email-list .email-options .settings-button-group {
-    position: absolute;
-    top: 0;
-}
-html[dir=ltr] .email-list .email-options .settings-button-group {
-    left: 0;
-}
-html[dir=rtl] .email-list .email-options .settings-button-group {
-    right: 0;
-}
-.email-list .email-options .settings-button-group .settings-button {
-    height: 35px;
-    min-width: 100px;
-    overflow: hidden;
-    text-align: left;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    width: 100px;
-}
-.email-list .email-options .settings-button-group .settings-button.set-primary {
-    margin-right: 10px;
-}
-@media screen and (min-width:400px) {
-    .email-list .email-options {
-        display: none;
-    }
-}
-.details {
-    color: #b1b1b3;
-    font-size: 12px;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    width: calc(95% - 95px);
-}
-.details.not-verified {
-    color: #d70022;
-}
-.details.verified {
-    color: #30e60b;
-    width: calc(95% - 195px);
-}
-.details > .di
```

Comment:
This might be a really terrible & hacky way to address #1101 in reusing styles from fxa-content-server, but it seems to work.